tools/authoring-decisions.adoc

```diff
@@ -511,9 +524,44 @@ We have the expression constraint which accepts a FEEL expression, the enumerati
 The last two create the equivalent FEEL expression as well.
 Depending on the base type, the enumeration and range will not be available.
 
+[id="included-models"]
 === C. Included models
 
-. TODO
+By default, there is no included model and you should see an empty state screen on the **Included models** tab.
+
+image::tools/authoringDecisions/included-models-tab-empty-state.png[Included models tab empty state]
+
+To include a first model we need to click **(1) Include model** button.
+Once the button was clicked a modal will appear.
+The modal allows you to include all **DMN** and **PMML** models from the current folder.
+
+image::tools/authoringDecisions/included-models-tab-modal.png[Include model modal]
+
+**(1) all models** is the current folder indicator.
+The **all models** is folder name used only for this demo.
+You can choose any folder name you need.
+
+**(2) Model** dropdown will list all **DMN** and **PMML** models from the **all models** folder.
+
+**(3) Name** is a namespace we give to the included model and this namespace will be used for referencing included nodes and data types.
+
+NOTE: With DMN 1.5. we can keep this empty to use the xref:authoring-decisions.adoc#import-to-default-namespace[default namespace] feature.
+
+**(4) Include model** is the button for confirming the include operation.
+
+Depending on the number of models you include, the **Included models** tab will change its visual somehow like below.
```

Comment:
Jason was talking about accessibility the other day, so maybe something a bit more descriptive might be better.
e.g.
Upon confirmation, the selected model(s) will be displayed below the **Included models** tab as a card.
WDYT?